pkg/apis/rook.io/v1/types.go

```diff
@@ -110,7 +110,7 @@ type StorageClassDeviceSet struct {
 	Config               map[string]string          `json:"config,omitempty"`               // Provider-specific device configuration
 	VolumeClaimTemplates []v1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"` // List of PVC templates for the underlying storage devices
 	Portable             bool                       `json:"portable,omitempty"`             // OSD portability across the hosts
-	TuneSlowDeviceClass  bool                       `json:"tuneDeviceClass,omitempty"`      // TuneSlowDeviceClass Tune the OSD when running on a slow Device Class
+	TuneSlowDeviceClass  bool                       `json:"tuneDeviceClass,omitempty"`      // TuneDeviceClass Tune the OSD when running on a slow Device Class
```

Comment:
This comment looks more like a description of a field than a JSON element.

Reply:
Indeed, fixed. Thank you for pointing it out!